src/GovUk.Education.ExploreEducationStatistics.Content.Search.FunctionApp/Extensions/ConfigurationExtensions.cs

```diff
@@ -1,10 +1,23 @@
 ﻿using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Hosting;
 using Serilog;
 
 namespace GovUk.Education.ExploreEducationStatistics.Content.Search.FunctionApp.Extensions;
 
 public static class ConfigurationExtensions
 {
+    public static IConfigurationBuilder SetBasePath(this IConfigurationBuilder builder, IHostEnvironment hostEnvironment)
+    {
+        Log.Logger.Information("Directory.GetCurrentDirectory() = {CurrentDirectory}", Directory.GetCurrentDirectory());
+        Log.Logger.Information("context.HostingEnvironment.ContentRootPath = {ContentRootPath}", hostEnvironment.ContentRootPath);
+        if (hostEnvironment.IsProduction())
+        {
+            Log.Logger.Information("Setting base path to: /home/site/wwwroot");
+            builder.SetBasePath("/home/site/wwwroot");
+        }
+
+        return builder;
+    }
```

Comment:
Have you seen this?
https://learn.microsoft.com/en-us/azure/azure-functions/functions-dotnet-dependency-injection#customizing-configuration-sources

It uses this as an example

```csharp
Path.Combine(context.ApplicationRootPath, $"appsettings.{context.EnvironmentName}.json")
```

Could try that as the first attempt, rather than using `SetBasePath`? Does it break the tests?